hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/endpoint/BucketEndpoint.java

```diff
@@ -348,12 +348,16 @@ public Response listMultipartUploads(
       AUDIT.logReadSuccess(buildAuditMessageForSuccess(s3GAction,
           getAuditParameters()));
       getMetrics().incListMultipartUploadsSuccess();
+      getMetrics().addListMultipartUploadsLatencyNs(
+          Time.monotonicNowNanos() - start, true);
```

Comment:
> I think we could simplify metrics by incrementing operation count and recording operation latency in the same method.

Let me clarify my previous comment with an example.

My suggestion is to change `incListMultipartUploadsSuccess()` to `listMultipartUploadsSuccess(long startNanos)`, and let it:
 * increment success counter
 * record elapsed time since `startNanos` as latency

This way we could easily ensure that:
 * all operations have both counter and latency
 * each operation records both count and latency together (avoid accidentally forgetting to do one or the other)

Reply:
Understood! Thank you for clarifying, will be making the required changes.